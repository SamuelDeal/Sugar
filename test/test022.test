#! echo 1
echo 2
//echo3
echo 4
// echo 5
echo 6 //echo 62
echo 7
/* echo 8 */ echo 82
echo 9
/* echo 10 
echo 11 */
echo 12
echo 13 /* echo 132 */
echo 14
# echo 15
echo 16 #    echo 17
echo 18
/** echo 19 
/* echo 20 */
echo 21
**/
echo 22

