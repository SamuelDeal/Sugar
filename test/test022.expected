2
4
6
7
82
9
12
13
14
16
18
22
